root/static/scripts/release-editor/fields.js

```diff
@@ -511,7 +515,8 @@ class Medium {
              );
     });
     this.confirmedFeatOnTrackTitles = ko.observable(
-      this.hasFeatOnTrackTitles.peek(),
+      !self.tracksUnknownToUser() &&
```

Comment:
Not sure I follow this bit: can `tracksUnknownToUser` ever be `true` here in the constructor? AFAICT it can only become true in `tracksLoaded`, or by the user ticking it.

Reply:
I wasn't sure about this either, so I was trying to play it safe (and lazy) by just copying over the expression used to initialize `hasFeatOnTrackTitles` above while switching it to use `hasFeatOnOrigTitle` rather than `hasFeatOnTitle`.

Does `tracksLoaded` always get called after the constructor when loading a medium from an existing release? If so, should `confirmedVariousArtists` and `confirmedFeatOnTrackTitles` both just be initialized to false here before being updated in `tracksLoaded`? (I think the intent was for the checkboxes to start out unchecked unless there are already tracks that possibly aren't following the guidelines, right?)

Reply:
I think you're right about this; I see the same issue with the VA checkbox being pre-checked when I seed a new release with a VA track credit. The checkboxes seem to work as intended both when seeding new releases and editing existing releases with bad data when I initialize both of these to false.

There's still a difference between the two, in that the feat. checkbox is only shown if the user introduced the feat. title while the VA checkbox is shown whenever there's a VA credit, even if it was already there . I think that that probably can't be changed without tracking more state about the original credits, though. I'd prefer to not introduce that in this change if that's okay. :-)